ownCloud/Viewer/DisplayViewController.swift

```diff
@@ -246,6 +254,15 @@ class DisplayViewController: UIViewController {
 		// This function is intended to be overwritten by the subclases to implement a custom view based on the source property.s
 	}
 
+	func hideUIElements() {
```

Comment:
@pablocarmu Can we use a better method name here? It is not hiding arbitrary UI elements, I think all these elements are related to certain context, right?